### PR TITLE
Feature/create hugo site

### PR DESCRIPTION
**Pull request by agnessnowplow**
### Initial draft PR only: 

Some changes still expected to come:
1. Tracking Overview - "React Notes from agnes" needs to be replaced and potentially relevant content changed.
2. Additional sample data upload method to be added (if works).

### How to test:
- Please install Hugo first:
``` brew install Hugo```
- Clone the repo
- Checkout this branch
- Add submodule that does not get cloned but it is needed
```git submodule update --init --recursive```
- Run `hugo serve`
- Check the results in your browser 
http://localhost:1313/advanced-analytics-web-accelerator/

### CODE REVIEW

File: content/upload/upload_1.md [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+```
+
+***
+
+#### **Step 5:**  Drop the **SAMPLE_EVENTS_BASE** table
+
+```sql
+DROP TABLE TARGET_DB.ATOMIC.SAMPLE_EVENTS_BASE
+```
+Now you should have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data.
```
Comment (Craig-Mace) [inline content/upload/upload_1.md:411]: Change to
"You will now have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data."

File: content/upload/upload_1.md [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+Schema created
+Staging table YOUR_DB.ATOMIC.SAMPLE_EVENTS_STAGED is created
+Stage dropped, if applicable
+Stage created
+File put to stage
+Data loaded into staging table
+Target table: YOUR_DB.ATOMIC.SAMPLE_EVENTS is created
+Staging table: YOUR_DB.ATOMIC.SAMPLE_EVENTS_STAGED is dropped
+```
+Now you should have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data.
```
Comment (Craig-Mace) [inline content/upload/upload_1.md:63]: Change to
"You will now have the ATOMIC.SAMPLE_EVENTS created and loaded with sample data."

File: content/upload/upload_1.md [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+
+3.1 Log into your web interface and click on `Databases` tab.
+
+3.2 Locate the **SAMPLE_EVENTS_BASE** table that you just created and select it.
+
+3.3 Click the `Load Data` button to open the Load Data wizard.
+
+3.4 Select the relevant warehouse from the dropdown list. Click `Next`.
+
+3.5 Within the `Source Files` section select `Load files from your computer` option, and click the `Select Files` button. If you have not saved the sample file provided as an attachment above please do so.
```
Comment (Craig-Mace) [inline content/upload/upload_1.md:242]: No need for the comma after "..... option, and click....."

File: content/upload/upload_1.md [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+
+3.7 Click the `Load` button (no need to alter the Load Options). Loading should take place within a couple of minutes.
+
+For more details please check out the official [Snowflake documentation](https://docs.snowflake.com/en/user-guide/data-load-web-ui.html).
+
+***
+
+#### **Step 4:** Create the **ATOMIC.SAMPLE_EVENTS** table
+
+The Snowplow pipeline creates context fields as arrays however for the web data model to work, these need to be converted to varchars. Run the below DDL statement in your SQL editor:
```
Comment (Craig-Mace) [inline content/upload/upload_1.md:257]: Small change here to add a comma, from "...as arrays however for..." to
"as arrays, however for"

**Review by Craig-Mace — COMMENTED** [review]
Small text and grammar changes

**Comment by agnessnowplow** [comment]
Thanks for the suggestions, Craig! Text and grammar changes made and pushed.

File: content/tracking/tracking_2.md [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+
+Next import the plugins to your `tracker.js` file.
+
+```javascript
+import { LinkClickTrackingPlugin, enableLinkClickTracking } from '@snowplow/browser-plugin-link-click-tracking';
+import { FormTrackingPlugin, enableFormTracking } from '@snowplow/browser-plugin-form-tracking';
+
+```
+
+Add the 2 plugins to your tracker instance.
```
Comment (Craig-Mace) [inline content/tracking/tracking_2.md:224]: Change the number 2 to "two"

File: content/tracking/tracking_2.md [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+
+Next import the plugins to your `snowplow.service.ts` file.
+
+```javascript
+import { LinkClickTrackingPlugin, enableLinkClickTracking } from '@snowplow/browser-plugin-link-click-tracking';
+import { FormTrackingPlugin, enableFormTracking } from '@snowplow/browser-plugin-form-tracking';
+
+```
+ 
+Add the 2 plugins to your tracker instance in `snowplow.service.ts` .
```
Comment (Craig-Mace) [inline content/tracking/tracking_2.md:272]: Change the number 2 to "two"

File: content/tracking/tracking_3.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
++++
+title = "Adding Context"
+weight = 3
+post = ""
++++
+
+Whilst the tracking set up provides event data on user **actions** at a specific point in time, **context** describes the setting in which an event takes place. To describe the context of an event, we need to define and capture individual **entities**. For example:
```
Comment (Craig-Mace) [inline content/tracking/tracking_3.md:7]: Change "set up" to "set-up"

File: content/tracking/tracking_3.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+- **webPage Entity -** Adds the Pageview ID 
+- **session Entity -** Information about the user session
+- **performanceTiming Entity -** Calculate page performance metrics
+- **geolocation Entity -** Information on the users location
+
+{{< tabs groupId="select" >}}
+{{% tab name="JS" %}}
+
+#### **Step 1:** Enable Predefined Entities
```
Comment (Craig-Mace) [inline content/tracking/tracking_3.md:25]: Change "Predefined Entities" to be all lower case to be consistent with other titles

File: content/next_steps/next_steps_1.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+At this stage you should:
+
+- Have tracking and enrichment setup
+- Have some data in the `ATOMIC.EVENTS` table
+- Enabled IAB, UA parser and YAUAA enrichments
+- Have a working dbt project with the web model configurations for the sample data
+
+#### **Step 1:** Complete refresh of your Snowplow web package (Optional)
+
+If you would like to use your current dbt environment that you set up during modelling the sample data you might want to start from scratch.
```
Comment (Craig-Mace) [inline content/next_steps/next_steps_1.md:16]: Change "set up" to "set-up"

File: content/next_steps/next_steps_2.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
++++
+title = "Visualise your pipeline data"
+weight = 2
+post = ""
++++
+
+#### **Visualise your pipeline data**
```
Comment (Craig-Mace) [inline content/next_steps/next_steps_2.md:7]: Recommend removing this line, as no other pages has it and is just repeating the page title.

File: content/next_steps/next_steps_2.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+
+Assuming you have already set up your Streamlit project all you have to do is change the schema within the queries used to generate the visualisations.
```
Comment (Craig-Mace) [inline content/next_steps/next_steps_2.md:9]: Change "set up" to "set-up"

**Review by Craig-Mace — COMMENTED** [review]
Sorry, few more now I've been through the site fully.

File: content/modeling/modeling_3.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+Run our recommended selector specified tests to identify potential issues with the data:
+
+```bash
+dbt test --selector snowplow_web_lean_tests
+```
+***
+#### **Step 3:** Explore your data
+
+Take some time to familiarise yourself with the derived tables. You could run a few simple queries such as the ones listed below:
```
Comment (Craig-Mace) [inline content/modeling/modeling_3.md:28]: Think we leave the SQL scripts to be showing always. No other page has this expand / contract for the code, so want to remain consistent. Shame really as I like this expand / contract.

File: content/visualisation/visualisation_1.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+#### **Step 1:** Install Requirements
+Run the command below to install the project requirements and run the virtual environment
+
+```bash
+pipenv install
+pipenv shell
+```
+
+#### **Step 2:** Setup Database Connection
+Open `secrets.toml` and add your Snowflake account and database details. **Ensure `secrets.toml` is in `.gitignore` to keep your information safe.**
```
[the remark this answers is not in the document]
Comment (agnessnowplow) [reply content/visualisation/visualisation_1.md:24]: Yes, good idea, I will add it into a warning block.
image

File: content/tracking/tracking_1.md [hunk 14]
```diff
@@ -0,0 +1,171 @@
++++
+title = "Setup"
```
Comment (Craig-Mace) [inline content/tracking/tracking_1.md:2]: Change to "Set-up"

File: content/modeling/modeling_3.md (hunk 12 above)
Comment (agnessnowplow) [reply content/modeling/modeling_3.md:28]: After discussing the issue with Jack he proposed another idea, there was a way to limit code blocs to a specific size and have the rest available by scrolling. It works both within and outside the tab so we can keep it consistent. I pushed the code as we both liked it so you can have a look at how it looks.

**Comment by Jack-Keene** [comment]
Closing out of date PR

**Pull request closed without merging**